docs/feature_combo.md

```diff
@@ -153,6 +157,7 @@ In order to use these features, the following configuration options and function
 | `COMBO_TERM_PER_COMBO`      | uint16_t get_combo_term(uint16_t index, combo_t \*combo)  | Optional per-combo timeout window. (default: `COMBO_TERM`)                                             |
 | `COMBO_MUST_HOLD_PER_COMBO` | bool get_combo_must_hold(uint16_t index, combo_t \*combo) | Controls if a given combo should fire immediately on tap or if it needs to be held. (default: `false`) |
 | `COMBO_MUST_TAP_PER_COMBO`  | bool get_combo_must_tap(uint16_t index, combo_t \*combo)  | Controls if a given combo should fire only if tapped within `COMBO_HOLD_TERM`. (default: `false`)      |
+| `COMBO_MUST_PRESS_IN_ORDER_PER_COMBO` | bool get_combo_must_press_in_order(uint16_t index, combo_t \*combo) | Controls if a given combo should fire only if its keys are pressed in order. (default: `true`) |
```

Comment:
Does that mean that in absence of any explicit configuration by the user, all combos must be pressed in order?

Reply:
Apparently this option is implemented in a way that it requires defining both `COMBO_MUST_PRESS_IN_ORDER` and `COMBO_MUST_PRESS_IN_ORDER_PER_COMBO` (without `COMBO_MUST_PRESS_IN_ORDER` the `COMBO_MUST_PRESS_IN_ORDER_PER_COMBO` define would just be ignored). Is this intended? Other per-key config options (e.g., from the tap/hold configuration) do not behave like that — defining just the per-key option and the corresponding function is enough.

Reply:
Ah, definitely not intended to be that way. Need to fix that.

Reply:
Right, docs fixed and the define checks fixed so that only the per combo variant is enough.

> Does that mean that in absence of any explicit configuration by the user, all combos must be pressed in order?

Yes. It's the same as if you'd just define `COMBO_MUST_PRESS_IN_ORDER`, it applies to all combos.